com/help/shell.d

```diff
@@ -20,10 +20,12 @@ Following is documentation for the PyIShellLink object.
 
 @ex This documentation class is based on:
 	http://msdn.microsoft.com/isapi/msdnlib.idc?theURL=/library/sdkdoc/shellcc/shell/ifaces/ishelllink/ishelllink.htm
+    (Updated docs at https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nn-shobjidl_core-ishelllinkw)
 	With only minor alterations and notations by Mike Fletcher.
 	Errors may be present, read at your own risk.
 See also:
 	http://msdn.microsoft.com/isapi/msdnlib.idc?theURL=/library/books/win95ui/chpt09-01.htm
+    (Archive found at https://library.thedatadungeon.com/msdn-2000-04/win95ui/html/chpt09-01.htm)
```

Comment:
I could not find these on the Internet Archive. However the first link seems pretty clear about documenting the IShellLink interface.

As for the second link, I found some archive of that digital book. Not sure if that documentation and samples are even still relevant.

Reply:
I can't believe the are, the see also could probably just be removed?

Reply:
Removed the second link there.
For the first link I took a look at the new docs and it should be easy to just rewrite the comments. I added a TODO like the other similar updated documentation links and will do a follow-up